includes/wikia/DefaultSettings.php

```diff
@@ -30,6 +30,13 @@
  */
 $wgUseFakeExternalStoreDB = false;
 
+/**
+ * @name wgEnableSwiftFileBackend
+ * Enables distributed file system
+ *
+ * TODO: will be removed fully by SUS-2429
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)